Library/Formula/ksh.rb

```diff
@@ -2,8 +2,12 @@
 class Ksh < Formula
   desc "ksh93, the KornShell"
   homepage "http://www.kornshell.com"
-  url "http://www2.research.att.com/~astopen/download/tgz/ast-ksh.2012-08-01.tgz",
-    :using => :nounzip, :user => "I accept www.opensource.org/licenses/eclipse:."
+  # Reported to admin@kornshell.com on 29th Feb 2016
+  # url "http://www2.research.att.com/~astopen/download/tgz/ast-ksh.2012-08-01.tgz",
+  #   :using => :nounzip, :user => "I accept www.opensource.org/licenses/eclipse:."
+  url "http://www.opensource.apple.com/source/ksh/ksh-23/ast-ksh.2012-08-01.tgz",
```

Comment:
Would you mind using `https://opensource.apple.com/` here? (note the HTTPS and removed `www.`)